libc/hdrgen/yaml/unistd.yaml

```diff
@@ -141,6 +141,12 @@ functions:
       - type: int
       - type: __getoptargv_t
       - type: const char *
+  - name: getpagesize
+    standards:
+      - POSIX
+    return_type: int
+    arguments:
+      - type: void
```

Comment:
Consider breaking getpagesize out into its own PR that's orthogonal to a new memory allocator.

Reply:
Will do, I didn't want PR's to get tied up depending on other PR's.

Reply:
Sorry, did you move it to a distinct PR? or not yet?